src/test/java/CommandParserTest.java

```diff
@@ -0,0 +1,90 @@
+import commands.AddDeadlineCommand;
+import commands.AddEventCommand;
+import commands.AddToDoCommand;
+import commands.DeleteCommand;
+import commands.MarkDoneCommand;
+import exceptions.MorganException;
+import org.testng.annotations.Test;
+import parser.CommandParser;
+import tasks.TaskList;
+import tasks.ToDoTask;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CommandParserTest {
+    private final CommandParser parser = new CommandParser();
+    private final Ui ui = new Ui();
+
+    @Test
+    public void testDelete() {
+        TaskList t1 = new TaskList();
```

Comment:
Perhaps the variable names for both tasks could have more appropriate names like ```expectedTaskList``` and ```actualTaskList```?